### ISSUE TITLE
Requesting to put the 'Cookie Preferences' icon in the right side.

### ISSUE THREAD

**Issue by RitamSamant** [cncf/glossary#1852]
I would like to request the team to allow me to put the 'Cookie Preferences' icon on the right side so that the icon doesn't get overlapped with the index contents.

Issue
![Screenshot (43)](https://user-images.githubusercontent.com/91908473/228279476-5b754c70-bfbd-4471-bed1-eec3d8817b94.png)

Solution
![Screenshot (40)](https://user-images.githubusercontent.com/91908473/228279809-6d2dcb04-21ee-453d-9ccc-09aa3efe552d.png)

If you think it will be useful to other users I would like to work on this. Thanks.

**Comment by seokho-son** [comment]
Hi @cjyabraham 
Is this request acceptable ?

**Comment by cjyabraham** [comment]
Hmm, I see what you're saying, however, the downside is that it will now be at a different location from [all other CNCF sites](https://www.cncf.io/all-cncf/). I think personally I'd prefer to keep common UI elements like this consistent across all sites.

**Comment by cjyabraham** [comment]
I don't think it will look good if it's more transparent and words can show through. It'll look a bit cluttered and messy. Is this really a problem? If words are behind the cookie, can't you just scroll up to see them?

**Comment by jihoon-seo** [comment]
This issue has been stale for the long term, and seems that not resolvable solely for the Glossary website without hurting the consistency among all CNCF websites.
IMO this suggestion makes sense enough.
If needed, we might open an issue in other repo, say in CNCF umbrella, that suggests to move the cookie icon to the right side.
Considering these aspects, let me close this issue in this repo.
